cmake/tribits/ci_support/filter-packages-list.py

```diff
@@ -46,30 +46,54 @@
 # Read in the commandline arguments
 #
 
-usageHelp = r"""filter-packages-list --input-packages-list=P1,P2,... --keep-types=T1,T2,...
+usageHelp = \
+r"""filter-packages-list.py --deps-xml-file=<PROJECT_DEPS_FILE> \
+  --input-packages-list=<P1>,<P2>,... --keep-test-test-categories=<T1>,<T2>,...
+
+This script takes in a comma-seprated list of TriBITS package names
+<P1>,<P2>,... and then filters out the package names for packages that don't
+the package test test categories listed in <T1>,<T2>,... which are given the
```

Comment:
"that don't the package test test categories" -- something happened to that sentence; not quite sure what word to fill in there.

Reply:
@mhoemmen, I fixed this in TriBITS 'master' as shown in the updated documentation at:

* https://tribits.org/doc/TribitsDevelopersGuide.html#filter-packages-list-py-help

I will update this in the Trilinos copy in a future Trilinos PR.